app/models/protected_area.rb

```diff
@@ -326,7 +328,10 @@ def self.with_valid_iucn_categories
     )
   end
 
+  OECM_ATTRS_ERROR = "'conservation_objectives' and 'supplementary_info can only be assigned if this is an OECM area".freeze
```

Comment:
I think there may be a `'` missing after supplementary_info?

Reply:
Well spotted! Fixed now. Thanks!